microcosm_logging/factories.py

```diff
@@ -113,20 +112,17 @@ def make_json_formatter(graph):
     Used for searchable aggregation.
 
     """
-    fields = {
-        # what was logged?
+    fields = """{
         "loggerName": "%(name)s",
         "levelName": "%(levelname)s",
-        "message": "%(message)s",
-        # when was it logged?
+        "message": %(message)s,
```

Comment:
On closer looking this does seem bad because this will *not* work properly for logs where message is a string (missing quotes). We probably need to add an extra field for `message_dict` so we can process it assuming it is a dict.